package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@ecp.eth/md",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.html",
+  "scripts": {
+    "dev": "vite ."
```

Comment:
do we need this file? think it's better to remove it

Reply:
just so that it is easier to run locally? i will remove for now